site/app/models/Config.php

```diff
@@ -24,7 +24,7 @@ class Config {
      * turn on if running server in production environment.
      * @var bool
      */
-    private $debug = __INSTALL__FILLIN__DEBUGGING_ENABLED__;
+    private $debug = false;
```

Comment:
@MasterOdin 
Please don't undo my earlier change.  This needs to be set in configure/install.